external/upstream/gau2grid/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+find_package(gau2grid CONFIG QUIET)  # TODO add version
+
+if(${gau2grid_FOUND})
+    get_property(_loc TARGET gau2grid::gg PROPERTY LOCATION)
+    message(STATUS "${Cyan}Found gau2grid${ColourReset}: ${_loc} (found version ${gau2grid_VERSION})")
+    add_library(gau2grid_external INTERFACE)  # dummy
+else()
+    if(${CMAKE_INSIST_FIND_PACKAGE_gau2grid})
+        message(FATAL_ERROR "Suitable gau2grid could not be externally located as user insists")
+    endif()
+
+    include(ExternalProject)
+    message(STATUS "Suitable gau2grid could not be located, ${Magenta}Building gau2grid${ColourReset} instead.")
+    ExternalProject_Add(gau2grid_external
+        DEPENDS pybind11_external
+        GIT_REPOSITORY https://github.com/loriab/gau2grid
```

Comment:
Merged your PR, feel free to redirect towards me when desired.

Reply:
done on this and omp below.